languages/en-ZA.js

```diff
@@ -0,0 +1,38 @@
+/*!
+ * numeral.js language configuration
+ * language : English for South Africa
+ * author : Stewart Scott : https://github.com/stewart42
+ */
+(function() {
+    var language = {
+        delimiters: {
+            thousands: ',',
+            decimal: '.'
```

Comment:
You are correct! However I suspect if it is changed to the 'correct' formatting, most developers will then override as per above, as that seems to be how _most_ software displays numbers in ZA. Thoughts?